api/tests/test_views.py

```diff
@@ -640,32 +640,6 @@ def test_aggregate_shift_content_handles_vacation_shifts(
         assert content["26.01.2019"]["type"] == "Vacation"
         assert content["26.01.2019"]["sick_or_vac_time"] == "04:00"
 
-    @pytest.mark.django_db
```

Comment:
Den Test sollten wir allerdings ersetzen gegen den des `lock_shifts` Endpoints. Erstelle dazu doch bitte gleich ein Issue

Reply:
Ist für den `lock_shifts` Endpunkt noch kein Test drin?

Reply:
Ach, tatsächlich doch https://github.com/ClockGU/clock-backend/blob/develop/api/tests/test_views.py#L324 . Also good to go. alles fein hier.